find.go

```diff
@@ -42,7 +52,7 @@ func (s *server) findMultihash(w http.ResponseWriter, r *http.Request) {
 		discardBody(r)
 		handleIPNIOptions(w, true)
 	case http.MethodPost:
-		s.find(w, r)
+		s.find(w, r, nil)
```

Comment:
doesn't this fail now since mh is nil?

Reply:
Right; I have removed the check that would have otherwise caused batch find to fail. Multihash is only used in the streaming path which only supports single multihash lookup.

Thank you for pointing this out.